app/src/main/java/com/example/myapplication/adapter/UsersAdapter.kt

```diff
@@ -1,35 +1,79 @@
 package com.example.myapplication.adapter
 
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
+import com.example.myapplication.R
 import com.example.myapplication.databinding.UserPatternBinding
 import com.example.myapplication.loadImage
 import com.example.myapplication.model.User
 
+private const val DEFAULT_MARGIN = 8
+private const val MULTISELECT_MODE_MARGIN = 40
 
 class UsersAdapter(private val actionListener: UserActionListener) :
     ListAdapter<User, UsersAdapter.UsersViewHolder>(MyItemCallback()) {
 
+    private val _isModeActive = MutableLiveData(false)
+    private val isModeActive: LiveData<Boolean> get() = _isModeActive
+
+    fun changeModeStatus(state: Boolean) {
+        _isModeActive.value = state
+    }
 
     class UsersViewHolder(
-        private val binding: UserPatternBinding, private val actionListener: UserActionListener
+        private val binding: UserPatternBinding,
+        private val actionListener: UserActionListener,
+        private val isModeActive: LiveData<Boolean>,
+        private val setModeActive: (Boolean) -> Unit
     ) : RecyclerView.ViewHolder(binding.root) {
+
         fun onBind(item: User) {
+            val context = binding.root.context
+            val density = context.resources.displayMetrics.density
 
-            with(binding) {
-                userNameView.text = item.name
-                userCareerView.text = item.career
-                loadImage(userPhotoView, item.photo)
+            isModeActive.observeForever { active ->
```

Comment:
First of all you are adding observer to observers list every time new viewholder is creating.
Next, you need to understand that adding an observer using the observeForever() function forces you to [manually remove](https://developer.android.com/reference/androidx/lifecycle/LiveData#observeForever(androidx.lifecycle.Observer%3C?%20super%20T%3E)) such an observer.